lib/controller.js

```diff
@@ -98,14 +102,14 @@ Tessel.list = function(opts) {
       } else if (foundTessels.length === 1) {
         // Close all opened connections and resolve
         controller.closeTesselConnections(foundTessels)
-          .then(resolve);
+          .then(() => resolve(foundTessels));
```

Comment:
For the sake of folks who want to do something like this (including the test rig):
```.js
t2.list().then((tessels) => {
...
});
```
It's much preferred to actually return the listed Tessels rather than nothing.